src/components/utils/test/thread_test.cc

```diff
@@ -49,20 +50,23 @@ const size_t kThreadStackSize = 32768u;
 const std::string kThreadName("Test thread");
 const uint32_t kWaitTime = 1000u;
 
-sync_primitives::ConditionalVariable cond_var_;
-sync_primitives::Lock test_mutex_;
+sync_primitives::ConditionalVariable cond_var_one;
+sync_primitives::ConditionalVariable cond_var_two;
```

Comment:
@AGritsevich The name is fine and it is following the Google coding style. Same goes for every other renaming.